smriprep/workflows/outputs.py

```diff
@@ -171,17 +168,16 @@ def init_anat_derivatives_wf(bids_root, freesurfer, num_t1w, output_dir,
     # CRITICAL: the sequence of labels here (CSF-GM-WM) is that of the output of FSL-FAST
     #           (intensity mean, per tissue). This order HAS to be matched also by the ``tpms``
     #           output in the data/io_spec.json file.
-    ds_std_tpms.inputs.extra_values = [f"label-{l}" for l in tpm_labels]
+    ds_std_tpms.inputs.label = tpm_labels
 
     # Transforms
     ds_t1w_tpl_inv_warp = pe.Node(
-        DerivativesDataSink(base_directory=output_dir, allowed_entities=['from', 'to', 'mode'],
-                            to='T1w', mode='image', suffix='xfm'),
+        DerivativesDataSink(base_directory=output_dir, to='T1w', mode='image', suffix='xfm'),
         name='ds_t1w_tpl_inv_warp', run_without_submitting=True)
 
     ds_t1w_tpl_warp = pe.Node(
-        DerivativesDataSink(base_directory=output_dir, allowed_entities=['from', 'to', 'mode'],
-                            mode='image', suffix='xfm', **{'from': 'T1w'}),
+        DerivativesDataSink(base_directory=output_dir, mode='image', suffix='xfm',
```

Comment:
Outstanding problem (coming from before this PR): how should we encode the cohort if several are used? perhaps with a `+` symbol, if bids-standard/bids-specification#226 (or part of it) finally goes forward?

like

```
sub-01_from-T1w_to-MNIPediatric+4_mode-image_xfm.h5
sub-01_from-T1w_to-MNIPediatric+5_mode-image_xfm.h5
```
where 4, and 5 indicate those cohorts.

Reply:
i thought that was solved (albeit hacky) before the PR by squishing together `space-MNIPediatric_cohort-1`? did we lose the ability to produce different cohorts / resolutions of the same template?

Reply:
sure, the `space_<id>_cohort-<n>` is not a problem. The problem comes when you want to write a transform file and there are several cohorts of the same template being used. In that situation, you'll need to cram the cohort field somewhere in the derivative name, and currently there is no way to do that.